src/Analysis/GWASV2/constants.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable import/prefer-default-export */
+export const gwasSteps = [
```

Comment:
I think we'll be able to remove this file once [VADC-216](https://ctds-planx.atlassian.net/browse/VADC-216) merges: since this workflow will only have one set of steps this information can be added directly in the JSX. Please let me know if there's more going on here - it relates to the work I have in progress.

Reply:
will swap to the place you specified in progress bar pr